### PR TITLE
reset password test fix

### PR DESCRIPTION
**Pull request by pondrejk**
not part of #11150 as this works ok in 6.12, the response format changed, hence the test failure

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_auth.py

**Review by ogajduse — APPROVED** [review]
PRT passed, ACK

File: tests/foreman/destructive/test_auth.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def test_positive_password_reset(target_sat):
     """
     result = target_sat.execute('foreman-rake permissions:reset')
     assert result.status == 0
-    reset_password = result.stdout.strip().split('password: ')[1]
+    reset_password = result.stdout.strip().split('\n')[0].split('password: ')[1]
```
Comment (Gauravtalreja1) [inline tests/foreman/destructive/test_auth.py:45]: Just to simplify this :)
```suggestion
    reset_password = result.stdout.splitlines()[0].split('password: ')[1]
```

Comment (shubhamsg199) [reply tests/foreman/destructive/test_auth.py:45]: Or `reset_password = result.stdout.splitlines()[0].split()[-1]`

**Review by shubhamsg199 — APPROVED** [review]
Ack

**Comment by pondrejk** [comment]
@Gauravtalreja1 thanks, suggestion committed, squash merge would be good in this case ;)

**Pull request merged**